package.json

```diff
@@ -22,7 +22,7 @@
   "dependencies": {
     "cassandra-driver": "^3.6.0",
     "inherits": "^2.0.3",
-    "lodash": "^4.17.10"
+    "lodash": "^4.17.11"
```

Comment:
Why not all the way to 4.17.14?

Reply:
Thanks for the catch (I had only bumped to meet the min CVE req). Fixed.